gstreamer-app-sys/Cargo.toml

```diff
@@ -1,45 +1,46 @@
+[badges.travis-ci]
+branch = "master"
+repository = "sdroege/gstreamer-sys"
+
 [build-dependencies]
 pkg-config = "0.3.7"
 
 [dependencies]
 libc = "0.2"
-glib-sys = { git = "https://github.com/gtk-rs/sys" }
 
-[dependencies.gstreamer-sys]
-path = "../gstreamer-sys"
+[dependencies.glib-sys]
+git = "https://github.com/gtk-rs/sys"
 
 [dependencies.gstreamer-base-sys]
 path = "../gstreamer-base-sys"
 
+[dependencies.gstreamer-sys]
+path = "../gstreamer-sys"
+
 [dev-dependencies]
 shell-words = "0.1.0"
 tempdir = "0.3"
 
 [features]
-v1_2 = []
-v1_4 = ["v1_2"]
-v1_6 = ["v1_4"]
-v1_8 = ["v1_6"]
-v1_10 = ["v1_8"]
+dox = []
+v1_10 = ["v1_6"]
 v1_12 = ["v1_10"]
 v1_14 = ["v1_12"]
-dox = []
```

Comment:
This needs some manual work. I'd suggest keeping the Cargo.toml as they were before with a simple `git checkout`.

Problem is that gir only adds version features for versions that have changes (for consistency I prefer to have them for all major versions), and doesn't add version features for minor releases (even if they add new API).